pystiche_papers/ulyanov_et_al_2016/_data.py

```diff
@@ -69,6 +73,22 @@ def forward(self, image: torch.Tensor) -> torch.Tensor:
         )
 
 
+def license_info(original: Optional[str] = None, repository: str = "ulyanov") -> str:
```

Comment:
If I'm not mistaken, `original` is never passed in this whole file. Given that `license` is a no-op in this case, why do we want to call it in the first place?